More localization parts

outloudvi opened the pull request:
Includes parts in menu items, card, member, music and event pages.

RayFirefist reviewed (changes requested):
I found out that you forgot to apply the translations even on cards detail page (unless it wasn't just committed now)

<img width="1680" alt="Screenshot 2021-04-14 at 16 28 20" src="https://user-images.githubusercontent.com/2178692/114727502-8af86580-9d3e-11eb-97ce-3051fb64cf70.png">

outloudvi commented:
> I found out that you forgot to apply the translations even on cards detail page (unless it wasn't just committed now)

Oh, I see! They are fixed now.

Also there are some changes that might not be able to convey through the title of this PR:
* I've changed the text layout of `/members`, making the subtitle of each character their English nickname (or say it, `FirstNameEnglish`), since the full English translation is already filled for the title.
* I've modified the layout of `/about` to make it more adorable.
